plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/MetastoreConfig.java

```diff
@@ -19,17 +19,31 @@
 public class MetastoreConfig
 {
     private boolean hideDeltaLakeTables;
+    private boolean hideIcebergTables;
 
     public boolean isHideDeltaLakeTables()
     {
         return hideDeltaLakeTables;
     }
 
+    public boolean isHideIcebergTables()
+    {
+        return hideIcebergTables;
+    }
+
     @Config("hive.hide-delta-lake-tables")
     @ConfigDescription("Hide Delta Lake tables in table listings")
     public MetastoreConfig setHideDeltaLakeTables(boolean hideDeltaLakeTables)
     {
         this.hideDeltaLakeTables = hideDeltaLakeTables;
         return this;
     }
+
+    @Config("hive.hide-iceberg-tables")
```

Comment:
I'm not sure whether the strategy of using `hive.hide-xxx-tables` properties is that good.
Would it make sense to rather offer a generic property to hide any non-hive tables?